### PR TITLE
Refined content and slides for Seurat DE

### PR DESCRIPTION
**Pull request by manveerchauhan**
Uploaded the latest slide deck I used during the workshop and adjusted afterward.

Also went through the markdown file and refined a couple things

### CODE REVIEW

File: docs/tutorials/seurat-de/seurat-de.md [hunk 1]
```diff
@@ -951,7 +940,6 @@ FeaturePlot(ifnb.filtered, reduction = 'umap.cca',
 # Find upregulated genes in each group (cell type and condition)
 ifnb.treatVsCtrl.markers <- FindAllMarkers(ifnb.filtered,
                                           only.pos = TRUE)
-saveRDS(ifnb.treatVsCtrl.markers, "ifnb_stimVsCtrl_markers.rds")
```
Comment (vinisalazar) [inline docs/tutorials/seurat-de/seurat-de.md:954]: I don't think this line should be removed. If learners are running this bit of code, it's helpful for them to save the data artifact or they would need to run it again later.

File: docs/tutorials/seurat-de/seurat-de.md [hunk 2]
```diff
@@ -741,11 +732,9 @@ after.harmony | after.seuratCCA
 !!! question
     What do you think of the integration results now?
 
-    **Challenge:** look at the pc1/2 plots for each integration method -\> explain different pca slots here
+    **Hint:** Also look at the pc1 and pc2 plots for each integration method
```
Comment (vinisalazar) [inline docs/tutorials/seurat-de/seurat-de.md:735]: Minor suggestion but capitalise PC1 and PC2 and add punctuation (period `.`) at the end of sentences.

**Review by vinisalazar — CHANGES_REQUESTED** [review]

**Comment by vinisalazar** [comment]
Also, Section 2 jumps from Step 3 to Step 5. Could you please fix that?

**Comment by manveerchauhan** [comment]
I agree with your suggestions - just implemented all of it. Closing this pull request, then opening another

**Pull request closed without merging**